bin/omarchy-cmd-screenshot

```diff
@@ -19,6 +19,24 @@ get_rectangles() {
   hyprctl clients -j | jq -r --arg ws "$active_workspace" '.[] | select(.workspace.id == ($ws | tonumber)) | "\(.at[0]),\(.at[1]) \(.size[0])x\(.size[1])"'
 }
 
+get_fullscreen_rect() {
+  local ws="$1"
+  # Try multiple field shapes for robustness across Hyprland versions:
```

Comment:
We should just make sure this works with 0.52. All Omarchy installs that'll get this new behavior will have the latest.

Reply:
Just retested on Omarchy 3.1.7 (Hyprland 0.52.0 via `hyprland --version`), and smart mode still auto-selects the fullscreen window without outlining hidden clients. I also refactored the helper to reuse the cached workspace and collapse the fullscreen jq fallback so the script matches the other omarchy-cmd-* tools. Let me know if you want extra logging, but this works on 0.52. Thanks!